.gitignore

```diff
@@ -1,5 +1,7 @@
+src/test/resources/shared
```

Comment:
putting the shared test files in the same place the shared common repository does for consistency and to make it easier to reuse testing tools